core/src/test/java/com/sequenceiq/cloudbreak/service/stack/StackServiceTest.java

```diff
@@ -223,7 +223,7 @@ public void testDeleteWhenStackCouldNotFindByItsNameForWorkspaceAndForcedAndDele
         expectedException.expect(NotFoundException.class);
         expectedException.expectMessage(String.format(STACK_NOT_FOUND_BY_NAME_MESSAGE, STACK_NAME));
 
-        underTest.deleteByName(STACK_NAME, WORKSPACE_ID, true, true, user);
```

Comment:
Some methods used deleteDeps with true, there the name of the test method also contains the DeleteDeps => they can be deleted from the name of the test method as well.